packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/index.js

```diff
@@ -27,6 +29,11 @@ export default function SidebarNavigationScreenNavigationMenus() {
 			PRELOADED_NAVIGATION_MENUS_QUERY
 		);
 
+	const firstNavigationMenu = navigationMenus?.[ 0 ];
+
+	const { handleSave, handleDelete, handleDuplicate } =
+		useNavigationMenuHandlers( firstNavigationMenu );
```

Comment:
An alternative which might make these more flexible is to require the `navigationMenu` argument to be supplied at _call_ time. This would allow you to use these callbacks for _any_ navigation rather than the one configured when the hook was initiated.

Reply:
Yes but we'd have the hook called when a navigation is already displayed because only then the UI to delete/duplicate/create appears. So 🤷🏻